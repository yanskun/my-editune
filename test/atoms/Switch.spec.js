import { shallowMount } from "@vue/test-utils"
import Component from "~/components/atoms/Switch"

describe("Switch", () => {
  test("snapshot true", () => {
    const wrapper = shallowMount(Component, {
      props: {
        value: true
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })

  test("snapshot false", () => {
    const wrapper = shallowMount(Component, {
      props: {
        value: false
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
