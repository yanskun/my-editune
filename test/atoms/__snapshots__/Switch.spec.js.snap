// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch snapshot false 1`] = `
<el-switch-stub
  activecolor=""
  activeiconclass=""
  activevalue="true"
  inactivecolor=""
  inactiveiconclass=""
  name=""
  validateevent="true"
  width="40"
/>
`;

exports[`Switch snapshot true 1`] = `
<el-switch-stub
  activecolor=""
  activeiconclass=""
  activevalue="true"
  inactivecolor=""
  inactiveiconclass=""
  name=""
  validateevent="true"
  width="40"
/>
`;
